core/domain/acl_decorators.py

```diff
@@ -922,11 +998,13 @@ def test_can_manage_topic_rights(self, topic_id, **kwargs):
 
         user_actions_info = user_services.UserActionsInfo(self.user_id)
 
-        if role_services.ACTION_EDIT_ANY_TOPIC in user_actions_info.actions:
+        if (
+                role_services.ACTION_CAN_MANAGE_TOPIC_RIGHTS in
```

Comment:
Sorry this should be ACTION_MANAGE_TOPIC_RIGHTS instead (for consistency)

Reply:
Done.